### DISCUSSION TITLE
How do I add this to the Shield plugin for the permissions?

### DISCUSSION THREAD

**Discussion by RoseRiyadh** [pxlrbt/filament-excel#218] [Q&A]
Hello, I hope you are doing well.
How do I add this to the Shield plugin for the permissions?

Thank you

**Comment by pxlrbt** [comment]
I think you can define custom permissions in Shield. Then you can add `->authorize(auth()->user()->can('your-permission'))` to the action

**Comment by RoseRiyadh** [comment]
it's not visible in shield, and the permission is added and it's not shown (as default it's not selected to it's not active). ran shield:generate and it didn't appear with the list of the shield permissions. what do you recommend??

**Comment by pxlrbt** [comment]
> it's not visible in shield, and the permission is added and it's not shown

You added it to the Shield config file? `entities.custom_permissions`?
